README.md

```diff
@@ -15,7 +15,7 @@ Dependencies
 Installing
 ----------
 - **debian** / **ubuntu**:
-  `apt-get install python3-yaml python3-requests python3-click python3-distro python3-psutil python3-pexpect`
+  `apt-get install python3-click, python3 -m pip install psutil, python3 -m pip install distro, python3 -m pip install pexpect, sudo python3 -m pip install pyyaml, sudo python3 -m pip install lib`
```

Comment:
we shouldn't need / use pip on debian; if it needs to be pip3; If the debian packages are not available on older releases, please add it as an alternative under a subject "older debian / ubuntu [version]"

Reply:
I revalidated the dependency on clean VM for Ubuntu 16.04, 18.04, 20.04 and found that it's not necessary to change any command for the time being.